Markdown/Markdown.sublime-syntax

```diff
@@ -8,14 +8,25 @@ file_extensions:
   - markdown
   - markdn
 scope: text.html.markdown
+variables:
+    separator_line: |-
+        (?x)
```

Comment:
I suggest always wrapping variables in non-capturing groups and multi-line expressions in `(?x:...)` groups (unless they represent a set). That way you can include them anywhere without having to worry about the including pattern being in extended more or not.

For example, in `contexts.main[1].match` you are already using extended mode, but if you were not the result could be different and unexpected, since extended mode would be active after you included this variable.
However, in `context.separator[0].match` you do not initially use extended mode, but if you were to use any spaces after the inclusion of this variable it would be ignored.

The reason why I wrap everything in a group is because it allows to treat variables atomicly. You can add quantifiers after variables and they mean exactly what you think they would (what the variable matches can be repeated) instead of potentially only repeating the last token of the variable's contents. For example, `{{block_quote}}+` (which makes no sense, but for the example's sake) would expand to `[ ]{,3}.+` and not `(?:[ ]{,3}>.)+`.